src/request-item/request-item.tsx

```diff
@@ -0,0 +1,54 @@
+import { h } from 'preact'
+import cn from 'classnames'
+import classes from './request-item.module.css'
+import defaultAvatar from './defaultAvatar.jpg'
```

Comment:
Would have preferred we keep using `kebab-case` for file names in general. But I let it fly for the icons, so I guess it's too little too late now. 😅

Reply:
Oh yeah I missed this but you're right. With #21 merged I can finish up my updates to the IconButton component, so since I'll be in there anyway I'll take this opportunity to rename the files and update the css. 😉